acme/acme/challenges.py

```diff
@@ -513,6 +513,27 @@ def validation(self, account_key, **kwargs):
         return self.response(account_key).gen_cert(key=kwargs.get('cert_key'))
 
 
+@ChallengeResponse.register
+class TLSALPN01Response(KeyAuthorizationChallengeResponse):
+    """ACME TLS-ALPN-01 challenge response.
+
+    This class only allows initiating a TLS-ALPN-01 challenge returned from the
+    CA. Full support for responding to TLS-ALPN-01 challenges by generating and
+    serving the expected response certificate is not currently provided.
+    """
+    typ = "tls-alpn-01"
+
+    def simple_verify(self, chall, domain, account_public_key):
```

Comment:
Unless you need or want this method on your side, I'd prefer we removed it and the tests for it. The only thing in the class definition would be the docstring and `typ`.

(If you're interested, the reason why I prefer this is `simple_verify` in our challenges has not worked well for us. The initial idea was we could do some sanity checking of challenges on our end before reaching out to the CA but there were a number of scenarios like weird firewall rules that would cause `simple_verify` to fail but  validation by the CA to succeed. For this reason, Certbot no longer uses it and we recommend other clients don't either. We've kept the methods around in some form of another to prevent API breakage, but this is a new challenge type so I'd prefer to skip on adding the method entirely until we have a need to do otherwise.)

Reply:
> Unless you need or want this method on your side, I'd prefer we removed it and the tests for it. The only thing in the class definition would be the docstring and typ.

WFM! Removed in 8b011a4. Thanks @bmw